src/layouts/Main/components/Sidebar/Sidebar.tsx

```diff
@@ -2,15 +2,15 @@ import React, { FC } from 'react';
 import clsx from 'clsx';
 
 import { makeStyles } from '@material-ui/styles';
-import { Divider, Drawer, Theme, DrawerVariant } from '@material-ui/core';
+import { Divider, Drawer, Theme } from '@material-ui/core';
 import DashboardIcon from '@material-ui/icons/Dashboard';
 import TableChartIcon from '@material-ui/icons/TableChart';
 import { Profile, SidebarNav } from './components';
 import useSelector from '../../../../hooks/useSelector';
 import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 import { userSelector } from '../../../../store/selectors/user';
 import { PlainFunction, Page } from '../../../../types/interfaces';
-
+import { DrawerVariant } from '../../../../types/types';
```

Comment:
why this change was needed? looks odd we need to re-create this interface in our project.

Reply:
Couldn't find a way to pull this specific type from their definition file. So i created a new type.
Is there any way to pull a specific type from interface?

Reply:
I would prefer doing this instead, as it seems very local to this use, and a bit arbitrary to have this in the general types file.
![image](https://user-images.githubusercontent.com/11048153/76138906-f1975400-6053-11ea-93ae-bb76a539a6ac.png)

Reply:
Agree, done